### ISSUE TITLE
ImageNet dataset no longer available 403 - Forbidden

### ISSUE THREAD

**Issue by andrewhand** [usnistgov/dioptra#19]
The following lines: 
```
# Download the ImageNet dataset
cd ${imagenet_data_dir}
kaggle competitions download -c imagenet-object-localization-challenge
unzip imagenet-object-localization-challenge.zip
tar -xzf imagenet_object_localization_patched2019.tar.gz
rm imagenet-object-localization-challenge.zip
rm imagenet_object_localization_patched2019.tar.gz
```
used in the the official docs [How to Obtain Common Datasets](https://github.com/usnistgov/dioptra/blob/main/docs/source/deployment-guide/obtaining-datasets.rst#:~:text=%23%20Download%20the%20ImageNet%20dataset,rm%20imagenet_object_localization_patched2019.tar.gz) now results in a 403 - Forbidden error probably due to the dataset no longer being available or changing ownership since this file was written. Terminal output:
```
403 - Forbidden
unzip:  cannot find or open imagenet-object-localization-challenge.zip, imagenet-object-localization-challenge.zip.zip or imagenet-object-localization-challenge.zip.ZIP.
tar: Error opening archive: Failed to open 'imagenet_object_localization_patched2019.tar.gz'
rm: imagenet-object-localization-challenge.zip: No such file or directory
rm: imagenet_object_localization_patched2019.tar.gz: No such file or directory
Downloading road-sign-detection.zip to /Users/ahand/Projects/data/kaggle_datasets/RoadSign-Kaggle-2019
```

**Comment by jkglasbrenner** [comment]
Figured it out, the 403 - Forbidden error appears if you don't agree to the competition rules (even though the competition is over). At the bottom of <https://www.kaggle.com/competitions/imagenet-object-localization-challenge/data>, you'll see this:

![image](https://user-images.githubusercontent.com/8547754/224135609-aca1823b-bb80-4d70-bf70-5db6da5f1b4f.png)

People need to sign-in using the account they used to generate the Kaggle API token, and then the "Sign In" will be replaced with I Accept/I Agree. Once you agree, that box gets replaced with:

![image](https://user-images.githubusercontent.com/8547754/224136267-2e3bc549-a088-4139-a19b-360a7ea0e013.png)

Now if you try to download using the kaggle CLI tool, it will work (provided you remember to generate the API token).

**Comment by jkglasbrenner** [comment]
Closed by #176
